.github/workflows/create-crd-update-pr-in-longhorn-repo.yml

```
@@ -0,0 +1,52 @@
+on:
+  pull_request:
+    branches: ["master"]
+    types:
+      - closed
+
+jobs:
+  if_merged:
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+    steps:
+    - name: Prepare Packages
```

yamllint (1.35.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)